src/components/Sections/AccordionSection/__tests__/AccordionSection.spec.ts

```diff
@@ -0,0 +1,174 @@
+import { render, fireEvent } from "@testing-library/vue";
+import AccordionSection from "..";
+
+describe("components/sections/accordion-section", () => {
+  describe("dark mode", () => {
+    it("renders a div with its class list containing Accordion--Dark when dark is set to true", () => {
+      const { container } = render(AccordionSection, {
+        props: {
+          dark: true,
+          items: [
+            {
+              title: "test",
+              text: "This is a test text to test text rendering",
+            },
+          ],
+        },
+      });
+      const darkAccordion = container.querySelector(".Accordion");
+      const classList = darkAccordion?.firstElementChild?.classList;
+      expect(classList).toContain("Accordion--Dark");
+    });
+    it("renders a div with its class list containing Accordion--Light when the dark property is set to false", () => {
+      const lightProps = {
+        dark: false,
+        items: [
+          {
+            title: "test",
+            text: "This is a test text to test text rendering",
+          },
+        ],
+      };
+
+      const { container } = render(AccordionSection, {
+        props: lightProps,
+      });
+      const lightAccordionClassList = container.querySelector(".Accordion")
+        ?.firstElementChild?.classList;
+
+      expect(lightAccordionClassList).toContain("Accordion--Light");
+    });
+    it("renders a div with its class list containing Accordion--Light when the dark property is not set", () => {
+      const darkNotSetProps = {
+        items: [
+          {
+            title: "test",
+            text: "This is a test text to test text rendering",
+          },
+        ],
+      };
+      const { container } = render(AccordionSection, {
+        props: darkNotSetProps,
+      });
+      const darkNotSetClassList = container.querySelector(".Accordion")
+        ?.firstElementChild?.classList;
+      expect(darkNotSetClassList).toContain("Accordion--Light");
+    });
+  });
+  describe("title prop", () => {
+    let container, accordionSection: HTMLElement;
+    beforeAll(() => {
+      container = render(AccordionSection, {
+        props: {
+          title: "This is a title",
+          items: [
+            {
+              title: "I love Luigi",
+              text: "Mario sux",
+            },
+          ],
+        },
+      }).container;
+      accordionSection = container.querySelector(
+        ".Accordion-Section",
+      )! as HTMLElement;
+    });
+    it("when the title prop is set the accordion-section has 3 children", () => {
```

Comment:
This test is too specific and will probably break, when the markup slightly changes. Please test for the appearance of the title text instead.